src/components/forms/FormSearchField.js

```diff
@@ -1,46 +1,96 @@
 /**
- * @author psarando
+ * @author psarando sriram
  */
 import React from "react";
+import { useTranslation } from "i18n";
 
-import Autocomplete from "../../components/autocomplete/Autocomplete";
+import Autocomplete, {
+    createFilterOptions,
+} from "@material-ui/lab/Autocomplete";
 
-import getFormError from "./getFormError";
-
-import FormControl from "@material-ui/core/FormControl";
-import FormHelperText from "@material-ui/core/FormHelperText";
+import FormTextField from "./FormTextField";
 
 const FormSearchField = ({
-    field: { value, onBlur, onChange, ...field },
+    field: { value, onChange, ...field },
     label,
     helperText,
     required,
-    form: { touched, errors, setFieldTouched, setFieldValue },
+    options,
+    size = "small",
+    renderCustomOption,
+    handleSearch,
+    form: { setFieldValue, ...form },
     ...props
```

Comment:
The dev console in Storybook is now showing these warnings:
```
Warning: React does not recognize the `labelKey` prop on a DOM element.
Warning: React does not recognize the `valueKey` prop on a DOM element.
```
These could be added to the destructuring here, then they won't be spread into the `FormTextField` below.